aea/protocols/generator/base.py

```diff
@@ -1991,6 +1993,9 @@ def generate_full_mode(self) -> None:
         # Run black formatting
         try_run_black_formatting(self.path_to_generated_protocol_package)
 
+        # Run isort formatting
+        try_run_isort_formatting(self.path_to_generated_protocol_package)
```

Comment:
this change is related to the protocol generator. We now call `isort` after `black`